.github/workflows/test.yml

```diff
@@ -100,3 +100,41 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - run: 'echo "No build required"'
+
+  e2e:
+    runs-on: ubuntu-latest
+    # dont run e2e tests on forked PRs
+    if: github.repository == 'runatlantis/atlantis'
+    env:
+      TERRAFORM_VERSION: 1.8.3
+      GITHUB_USERNAME: ${{ secrets.ATLANTISBOT_GITHUB_USERNAME }}
+      GITHUB_PASSWORD: ${{ secrets.ATLANTISBOT_GITHUB_PASSWORD }}
```

Comment:
Are there technical reasons for using basic auth over a PAT with a limited scope? Not blocking here, just curious - the new PAT is pretty flexible in terms of reducing the scopes down to specific permissions and repos